program/application/Calulate.java

```diff
@@ -0,0 +1,20 @@
+package application;
+
+import java.util.Map;
+
+public class Calulate {
+    private double offerValue;
+
+    public double getOfferValue() {
+        return offerValue;
+    }
+
+    public void calculateOffer(Suitcase suitcase, double pickSuitcase, int round){
```

Comment:
since you only use this class for calculation, this method could have also been just static